src/sentry/grouping/strategies/configurations.py

```diff
@@ -191,7 +191,9 @@ def register_strategy_config(id, **kwargs):
     base="newstyle:2019-10-29",
     risk=RISK_LEVEL_MEDIUM,
     changelog="""
-        * Initial version
+        * We added new language/platform specific stack trace grouping enhancements rules
```

Comment:
```suggestion
        * Added new language/platform specific stack trace grouping enhancements rules
```
to match the grammar of the other entries